src/app/_components/client/UserListItem/hooks/useToggleFollow/useToggleFollow.ts

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+
+interface UseToggleFollowProps {
+  isFollowing: boolean;
+  isPrivate: boolean;
+  handleToggleOn: () => void;
+}
+
+const useToggleFollow = ({ isFollowing, isPrivate, handleToggleOn }: UseToggleFollowProps) => {
+  const [isFollow, setIsFollow] = useState(isFollowing);
+  const [isPrivateFollow, setIsPrivateFollow] = useState(false);
```

Comment:
```suggestion
  const [isPrivateFollow, setIsPrivateFollow] = useState(false);
```

해당 상태도 추후 Api 연결시 외부에서 초기 상태를 받게 될까요??

페이지를 새로고침하면 팔로우 요청중이 유지가 안될것같아서요!

Reply:
아하 !
제가 테스트하다 보니 임의의 값을 넣어놨네요 변경하도록 하겠습니다 !! 👍

Reply:
생각해 보니 해당 데이터도 API 초기 데이터에 포함되어야 할 것 같네요 ! ㅎㅎ 😀